src/fields/mod.rs

```diff
@@ -155,7 +155,7 @@ pub trait CodecFieldElement:
 /// [1]: https://datatracker.ietf.org/doc/html/draft-google-cfrg-libzk-01#section-6.6
 /// [2]: https://en.wikipedia.org/wiki/Lagrange_polynomial#Definition
 /// [3]: https://github.com/abetterinternet/zk-cred-longfellow/issues/40
-pub trait LagrangePolynomialFieldElement: FieldElement {
+pub trait LagrangePolynomialFieldElement: CodecFieldElement {
```

Comment:
I'm not sure if we can make this change, because we plan to implement the NTT on $\mathbb{F}_\text{P-256}^2$, but not define a serialization format for it.

Reply:
Yep, good point. I think instead I'll define a supertrait over `LagrangePolynomialFieldElement + CodecFieldElement` and use that. But I don't know what I would call it, so for now I'll put the trait bounds back as they were and punt on the matter.